src/js/components/picklist/index.js

```diff
@@ -74,6 +75,10 @@ const PicklistItem = View.extend({
 
     if (this.model === this.state.get('selected')) classNames.push('is-selected');
 
+    if (this.getOption('styleForNav')) {
+      classNames[indexOf(classNames, 'picklist__item')] = 'picklist__item-nav';
+    }
```

Comment:
If the `styleForNav: true` option is passed to the droplist, the default classname of `picklist__item` is replaced with `picklist__item-nav` for each item in the picklist.

This allows me to add special styling for those picklist items used only in the nav menu.

Default `picklist__item` styling:

![Screenshot 2023-01-17 at 1 58 51 PM](https://user-images.githubusercontent.com/35355575/212999455-aa04d46b-4034-4f8a-95e4-f4f96c433a69.png)

`picklist__item-nav` styling used for nav menu droplists:

![Screenshot 2023-01-17 at 11 27 59 AM](https://user-images.githubusercontent.com/35355575/212999546-c1cfe901-c9cc-4b37-87d8-6ec9d2c25745.png)
![Screenshot 2023-01-17 at 2 00 11 PM](https://user-images.githubusercontent.com/35355575/212999688-c3b397b8-ab37-484b-88d8-8f60eb58160e.png)

Reply:
I think you can set an `itemClassName` here which would just override any values on `. picklist__item`. And so you'd probably want to add this new class to the `.app-nav__` block / file

Reply:
Yup, that's a much better way to do that. Updated to do that in a recent push.